ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/vlans.yml

```diff
@@ -22,6 +22,12 @@ vlans:
     private_vlan:
       type: isolated
       primary_vlan: 110
+  - id: 201
+    tenant: Tenant_B
+    name: ETREE_LEAF
+    e_tree:
+      leaf_role: true
```

Comment:
Do we expect other roles? If so `role: leaf` with valid_values (only one value for now) would be good(?)

Reply:
There are only two, you are either the root (default) or you are a leaf